manual/manual.adoc

```diff
@@ -665,3 +660,33 @@ number of cheats that you can turn to:
 | **IDBEHOLDM** | Gives the tactical survey map.
 | **IDBEHOLDL** | Gives the night vision goggles.
 |==========================
+
+<<<
+
+== Contributing to Freedoom ==
+
+Freedoom is a 
+https://en.wikipedia.org/wiki/Free_and_open-source_software[free and open source software (FOSS)]
+project contributed to by many users around the world. It is available as 
+both free in cost (free as in free beer) and in modification and 
+redistribution rights (free as in free speech) to end users,
+provided that https://github.com/freedoom/freedoom/blob/master/COPYING.adoc[certain requirenments]
```

Comment:
"The licensing terms" rather than a link to the GitHub page should be preferable. A hyperlink to a web resource may be fragile (the file might be renamed, we might change hosts, etc).

Reply:
Removed link and replaced with: "provided that the original software license is included and/or viewable in modified or redistributed versions" 6da5dc8362055e9ae6594a89b1cc1386ce856190